test/SIL/Serialization/deserialize_stdlib.sil

```diff
@@ -1,5 +1,5 @@
 // Make sure that we can deserialize the stdlib.
-// RUN: %target-sil-opt %platform-module-dir/Swift.swiftmodule > /dev/null
-// RUN: llvm-bcanalyzer %platform-module-dir/Swift.swiftmodule | %FileCheck %s
+// RUN: %target-sil-opt $(if [[ -d %platform-module-dir/Swift.swiftmodule ]]; then echo %platform-module-dir/Swift.swiftmodule/%target-swiftmodule-name; else echo %platform-module-dir/Swift.swiftmodule; fi) -module-name Swift > /dev/null
```

Comment:
Can we split this up someway please?  This will break with Windows, as the subshell syntax is not portable.

Reply:
Hm. It may just need another substitution, then.